src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/DefaultDocumentResolver.cs

```diff
@@ -45,25 +44,15 @@ public override bool TryResolveDocument(string documentFilePath, out DocumentSna
             _foregroundDispatcher.AssertForegroundThread();
 
             var normalizedPath = _filePathNormalizer.Normalize(documentFilePath);
-            if (!_projectResolver.TryResolvePotentialProject(normalizedPath, out var potentialProject))
+            if (!_projectResolver.TryResolveProject(normalizedPath, out var project))
```

Comment:
This check now goes through both the "potential" project and the MISC project for us as the `enforceDocumentInProject = true` by default.